src/test-e2e/cloud-foundry/cf-level/create-org-e2e.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { by, element, promise, protractor } from 'protractor';
+
+import { e2e } from '../../e2e';
+import { CFHelpers } from '../../helpers/cf-helpers';
+import { ConsoleUserType, E2EHelpers } from '../../helpers/e2e-helpers';
+import { CfTopLevelPage } from './cf-top-level-page.po';
+import { OrgFormPage } from './org-form-page.po';
+
+describe('Create Organization', () => {
```

Comment:
I think these tests are fine. We have some tests already in `organizations-spaces-e2e.spec.ts` which covers creating orgs + spaces in a combined way (like a new space in a new org). I would however pull the `Should validate org name` and `should show the Organization CLI commands` tests from there into here